app/helpers/application_helper.rb

```diff
@@ -44,9 +44,13 @@ def markdown(text)
     markdown = Redcarpet::Markdown.new(html_render, options)
     markdown.render(text).html_safe
   end
-  
+
   def max_width
-    "mw-xl"
+    if controller.controller_name == "lines"
+      "mx-auto"
```

Comment:
この`max_width` は `application.html.erb` で使用しています。構造を確認してみてください。

```suggestion
      "mw-md"
```

Reply:
・max_width がapplication.html.erb で使用されていることは確認しており、
それに対しては、else後の "mw-xl"をそのまま適用できるようにしました。

・こちらmw-mdにすると、中央揃えが効かなくなりテキストが少し左寄りになってしまう為、
あえてmx-autoにしました。
→最大幅のメソッドなので、それに合わせてmw-mdを使用したかったのですが。

誤って考えてるかもしれません。コメント頂けると有難いです。